addons/uom/models/uom_uom.py

```diff
@@ -93,6 +93,21 @@ def _check_category_reference_uniqueness(self):
             if uom_data['uom_count'] > 1:
                 raise ValidationError(_("UoM category %s should only have one reference unit of measure.") % (self.env['uom.category'].browse(uom_data['category_id']).name,))
 
+    @api.onchange('rounding')
+    def _onchange_rounding(self):
+        precision = self.env.ref('product.decimal_product_uom').digits
```

Comment:
The given XML id may not exist.  This one is safer:
```suggestion
        precision = self.env['decimal.precision'].precision_get('Product Unit of Measure')
```